app/src/androidTest/java/com/github/sdp_begreen/begreen/fragments/SendPostFragmentTest.kt

```diff
@@ -209,16 +222,22 @@ class SendPostFragmentTest {
                 imageView.setImageBitmap(postImage)
             }
 
-            onView(withId(R.id.post_category))
-                .perform(typeText("category"), closeSoftKeyboard())
+            // Pick a trash category at random
```

Comment:
This code is a duplicate of the previous, I suggest you to separate it into a private helper method.